bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/widgets/Display.java

```diff
@@ -1188,6 +1188,24 @@ static Image createIcon (Image image) {
 	return Image.win32_new (device, SWT.ICON, hIcon);
 }
 
+long getTextSearchIcon(int size) {
+	if (!sizeToSearchIconHandle.containsKey(size)) {
+		int searchIconResource = textUseDarkthemeIcons ? Text.IDI_SEARCH_DARKTHEME : Text.IDI_SEARCH;
+	    long iconHandle = OS.LoadImage (OS.GetLibraryHandle (), searchIconResource, OS.IMAGE_ICON, size, size, 0);
```

Comment:
You should check for `0` like `Test::addIcons()` was doing, in case there are no more handles,

Reply:
Done